src/front-cover.jl

```diff
@@ -38,208 +38,139 @@ colorSides =  vec(valsSides[end,:])
 Return the Julia Data Science book front cover.
 """
 function front_cover()
-    CairoMakie.activate!()
+    GLMakie.activate!()
```

Comment:
Do we need to change back to CairoMakie? @lazarusA

Reply:
I would say yes.